weave/wandb_interface/wandb_api.py

```diff
@@ -0,0 +1,449 @@
+# Official interface for interacting with the W&B API. All
```

Comment:
This is all legacy

Reply:
yeah, it's moved in the next pr:
https://github.com/wandb/weave/pull/2471